source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,52 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-04-21
+
+### Highlights
+
+#### Apps Framework (Developer Preview)
+ - This feature is available on Mobile Apps when Apps Framework becomes available on Mattermost v5.35.0 and the proxy plugin is loaded on an instance. The launch for the Developer Preview of the Apps Framework is scheduled for April 29th.
+
+#### Files Search
+ - Searching in Mattermost now finds both relevant messages and files in your team's conversation history. Search will return results for attachments that match the file name or contain matching text content within supported document types. [Learn more](https://mattermost.com/blog/file-search/).
+
+#### Granular Access to System Console Pages
+ - Migrated Experimental, About, Reporting, Environment, Site Configuration, Authentication, Integrations, Compliance sections to their respective sub-section permissions.
```

Comment:
@sbishel 

We should mention here that upgrading to this version will cause problems if the user decides to downgrade to an older version of Mattermost due to the permission deletion. 
As we discussed in our meeting, the best course of action is coming up with script that does the rollback.

Anyways just mentioning you on this to give visibility if there is anymore you'd like to add.

Reply:
I'll add a note to v5.35 changelog, likely not needed for the Cloud changelog.

Reply:
We reverted that change, so it is not needed in v5.35 change log either.